deployment/puppet/Puppetfile

```diff
@@ -1,17 +1,16 @@
 forge "https://forgeapi.puppetlabs.com"
 
 mod 'puppet/nodejs', '2.1.0'
-mod 'puppetlabs-apache', '1.11.0'
-mod 'puppetlabs/apt', '2.3.0'
-mod 'puppetlabs/docker_platform', '2.2.1'
-mod 'puppetlabs/limits', '0.1.0'
-mod 'puppetlabs/lvm', '0.7.0'
-mod 'puppetlabs/postgresql', '4.9.0'
-mod 'puppetlabs/rabbitmq', '5.6.0'
-mod 'puppetlabs/stdlib', '4.13.1'
-mod 'puppet-archive', '1.3.0'
-mod 'saz/timezone', '3.2.0'
-mod 'stankevich/python', '1.9.8'
-mod 'thias/sysctl', '1.0.2'
+mod 'puppet/python', # https://github.com/voxpupuli/puppet-python/issues/470
```

Comment:
This was fixed in the latest version: https://forge.puppet.com/puppet/python/changelog

Reply:
FYI, it was released 6 days ago. https://github.com/voxpupuli/puppet-python/releases

Reply:
Yes. We should still be able to use it, right?